pkg/test/framework/resource/context.go

```diff
@@ -20,6 +20,11 @@ type Context interface {
 	// cleaned up.
 	TrackResource(r Resource) ID
 
+	// GetResource find resource(s) of a given type and assigns them to the given reference.
```

Comment:
So the options for the type are really just `*SomeTypeThatExtendsResource` or `[] *SomeTypeThatExtendsResource}`, correct? Could we make that more clear in the docs?

Reply:
Let me know how it looks now